code/src/java/pcgen/gui2/PCGenStatusBar.java

```diff
@@ -62,19 +66,18 @@
 		this.messageLabel = new JLabel();
 		this.progressBar = new JProgressBar();
 		this.loadStatusButton = new Button();
-		initComponents();
-	}
 
-	private void initComponents()
-	{
 		setLayout(new BoxLayout(this, BoxLayout.LINE_AXIS));
 		add(messageLabel);
 		add(Box.createHorizontalGlue());
 		progressBar.setStringPainted(true);
 		progressBar.setVisible(false);
 		add(progressBar);
 		add(Box.createHorizontalGlue());
-		add(GuiUtility.wrapParentAsJFXPanel(loadStatusButton));
+		JFXPanel wrappedButton = GuiUtility.wrapParentAsJFXPanel(loadStatusButton);
+		//todo: calculate this rather than hard code
+		wrappedButton.setMaximumSize(new Dimension(750, 20000000));
```

Comment:
I think we should calculate this before landing this PR

Reply:
@karianna I'm not sure this is going to get handled. Are we going to merge?